Examples/BookingSystem.AspNetCore/Stores/OrderStore.cs

```diff
@@ -449,21 +449,33 @@ public override async Task<Order> GetOrderStatus(OrderIdComponents orderId, Simp
             if (getOrderResult == FakeDatabaseGetOrderResult.OrderWasNotFound) throw new OpenBookingException(new UnknownOrderError());
 
             var orderIdUri = RenderOrderId(dbOrder.OrderMode == OrderMode.Proposal ? OrderType.OrderProposal : dbOrder.OrderMode == OrderMode.Lease ? OrderType.OrderQuote : OrderType.Order, new Guid(dbOrder.OrderId));
-            var orderItems = dbOrderItems.Select((orderItem) => new OrderItem
+            var orderItems = dbOrderItems.Select((orderItem) =>
             {
-                Id = dbOrder.OrderMode != OrderMode.Lease ? RenderOrderItemId(OrderType.Order, new Guid(dbOrder.OrderId), orderItem.Id) : null,
-                AcceptedOffer = new Offer
+                var hasAttendeeDetails = orderItem.AttendeeEmail != null || orderItem.AttendeeGivenName != null
+                || orderItem.AttendeeFamilyName != null || orderItem.AttendeeTelephone != null;
+                return new OrderItem
                 {
-                    Id = orderItem.OfferJsonLdId,
-                    Price = orderItem.Price
-                },
-                OrderedItem = orderItem.OpportunityJsonLdId,
-                OrderItemStatus =
-                            orderItem.Status == BookingStatus.Confirmed ? OrderItemStatus.OrderItemConfirmed :
-                            orderItem.Status == BookingStatus.CustomerCancelled ? OrderItemStatus.CustomerCancelled :
-                            orderItem.Status == BookingStatus.SellerCancelled ? OrderItemStatus.SellerCancelled :
-                            orderItem.Status == BookingStatus.Attended ? OrderItemStatus.AttendeeAttended :
-                            orderItem.Status == BookingStatus.Absent ? OrderItemStatus.AttendeeAbsent : (OrderItemStatus?)null
+                    Id = dbOrder.OrderMode != OrderMode.Lease ? RenderOrderItemId(OrderType.Order, new Guid(dbOrder.OrderId), orderItem.Id) : null,
+                    AcceptedOffer = new Offer
+                    {
+                        Id = orderItem.OfferJsonLdId,
+                        Price = orderItem.Price
+                    },
+                    OrderedItem = orderItem.OpportunityJsonLdId,
+                    OrderItemStatus =
+                                orderItem.Status == BookingStatus.Confirmed ? OrderItemStatus.OrderItemConfirmed :
+                                orderItem.Status == BookingStatus.CustomerCancelled ? OrderItemStatus.CustomerCancelled :
+                                orderItem.Status == BookingStatus.SellerCancelled ? OrderItemStatus.SellerCancelled :
+                                orderItem.Status == BookingStatus.Attended ? OrderItemStatus.AttendeeAttended :
+                                orderItem.Status == BookingStatus.Absent ? OrderItemStatus.AttendeeAbsent : (OrderItemStatus?)null,
+                    Attendee = hasAttendeeDetails ? new Person
+                    {
+                        GivenName = orderItem.AttendeeGivenName ?? null,
```

Comment:
Not sure `?? null` has any affect here? If it's already null, then null will be assigned